miden-lib/asm/kernels/transaction/main.masm

```diff
@@ -36,152 +36,94 @@ const.EPILOGUE_END=131081
 # MAIN
 # =================================================================================================
 
-#! This is the entrypoint for the transaction kernel program. It is composed of the following
-#! program sections:
+#! Transaction kernel program.
 #!
-#! 1. Prologue: execute the transaction prologue which prepares the transaction for processing
-#!    by parsing the transaction data and setting up the root context.
-#! 2. Note Processing: execute the note processing loop which consumes each input note and
-#!    invokes the note script of each note via a `dyncall` instruction invocation.
-#! 3. Transaction Script Processing: execute the transaction script if it exists via the invocation
-#!    of a `dyncall` instruction.
-#! 4. Epilogue: execute the transaction epilogue which finalizes the transaction by computing the
-#!    created notes commitment, the final account hash, asserting asset invariant conditions and
-#!    asserting the nonce rules are upheld.
+#! This is the entry point of the transaction kernel, the program will perform the following
+#! operations:
 #!
-#! Stack:        [BH, acct_id, IAH, NC]
-#! Advice stack: [NR, PH, CR, SR, BR, PH, BN,
```

Comment:
This was out-of-date, the `NOTE_ROOT` is much deeper in the advice stack:

https://github.com/0xPolygonMiden/miden-base/blob/74a5371cb27c0d491af110a5da1dc432c044cf89/miden-lib/asm/miden/kernels/tx/prologue.masm#L920-L935

Instead of copying and pasting the docs, I added a reference to `prepare_transaction`